### DISCUSSION TITLE
Is OpenToFu Adoption Required in Terragrunt?

### DISCUSSION THREAD

**Discussion by gitsstewart** [gruntwork-io/knowledge-base#774] [How do I]
Hello,

My name is Anastasia and work as an SRE for a Spanish startup. Few months ago we decided to use Terragrunt for our IaC and right now we have concerns about the future of the tool regarding everything that is happening with Terraform lisensing. 

Right now we blocked our Terraform version to 1.5.5: are we defenetely be forced to use OpenTofu in order to continue using Terragrunt?

Thank you

Anastasia Kondratieva
Site Reliability Engineer

---

<ins datetime="2023-10-05T15:13:48Z">
  <p><a href="https://support.gruntwork.io/hc/requests/110507">Tracked in ticket #110507</a></p>
</ins>

**Comment by denis256** [comment]
Hello,
since Terragrunt is a wrapper, so far approach is to support both - `terraform` and `tofu`, you can continue using Terraform without any need for immediate action.
